unlockfps_nc/Model/Config.cs

```diff
@@ -26,6 +26,7 @@ public class Config
         public int CustomResY { get; set; } = 1080;
         public int MonitorNum { get; set; } = 1;
         public int Priority { get; set; } = 3;
+        public string CustomOption { get; set; } = "";
```

Comment:
```suggestion
        public string AdditionalCommandLine { get; set; } = "";
```
'CustomOption' seems to be ambiguous about what it does